common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -954,9 +954,11 @@ def test_user_can_navigate_to_members_profile_page(self):
         self._set_team_configuration_and_membership()
         self.team_page.visit()
 
+        learner_name = self.team_page.first_member_username
+
         self.team_page.click_first_profile_image()
 
-        learner_profile_page = LearnerProfilePage(self.browser, self.team_page.first_member_username)
+        learner_profile_page = LearnerProfilePage(self.browser, learner_name)
```

Comment:
@efischer19 The [master build right after this was merged in](https://build.testeng.edx.org/job/edx-platform-bok-choy-master/527/) failed on this line. And as the testcase class is decorated with at-flaky that means it failed twice in a row. [Here is the console output.](https://build.testeng.edx.org/job/edx-platform-test-subset/169772/console) You can see the screenshot there too, etc. if you click on Status to show the artifacts. :(
Subsequent builds have passed, so it looks like a flakiness issue.